### ISSUE TITLE
does not work with karma and should@8.0.x

### ISSUE THREAD

**Issue by Nr9** [shouldjs/sinon#3]
Does not work with Karma since shouldjs version 8. (it worked fine with version 7)

I get the following error: 
```
Uncaught TypeError: Cannot read property 'add' of undefined
```

**Comment by btd** [comment]
Can you provide complete test case?

**Comment by Nr9** [comment]
I created a PR for the test case.

**Comment by btd** [comment]
It is fixed in 8.1.1, thanks for great test case, it was easy to run.

**Comment by Nr9** [comment]
Pleasure!

Thank you for the fix.

Julien
